temporalio/worker/_nexus.py

```diff
@@ -0,0 +1,477 @@
+
+    async def _handle_start_operation_task(
+        self,
+        task_token: bytes,
+        start_request: temporalio.api.nexus.v1.StartOperationRequest,
+        headers: Mapping[str, str],
+    ) -> None:
+        """
+        Handle a start operation task.
+
+        Attempt to execute the user start_operation method and invoke the data converter
+        on the result. Handle errors and send the task completion.
+        """
+
+        try:
+            start_response = await self._start_operation(start_request, headers)
+        except BaseException as err:
+            logger.exception("Failed to execute Nexus start operation method")
+            handler_err = _exception_to_handler_error(err)
+            completion = temporalio.bridge.proto.nexus.NexusTaskCompletion(
+                task_token=task_token,
+                error=await self._handler_error_to_proto(handler_err),
+            )
+            if isinstance(err, concurrent.futures.BrokenExecutor):
+                self._fail_worker_exception_queue.put_nowait(err)
+        else:
+            completion = temporalio.bridge.proto.nexus.NexusTaskCompletion(
+                task_token=task_token,
+                completed=temporalio.api.nexus.v1.Response(
+                    start_operation=start_response
+                ),
+            )
+
+        try:
+            await self._bridge_worker().complete_nexus_task(completion)
+        except Exception:
+            logger.exception("Failed to send Nexus task completion")
+        finally:
+            try:
+                del self._running_tasks[task_token]
+            except KeyError:
+                logger.exception("Failed to remove completed Nexus operation")
+
+    async def _start_operation(
+        self,
+        start_request: temporalio.api.nexus.v1.StartOperationRequest,
+        headers: Mapping[str, str],
+    ) -> temporalio.api.nexus.v1.StartOperationResponse:
+        """
+        Invoke the Nexus handler's start_operation method and construct the StartOperationResponse.
+
+        OperationError is handled by this function, since it results in a StartOperationResponse.
+
+        All other exceptions are handled by a caller of this function.
+        """
+        ctx = StartOperationContext(
+            service=start_request.service,
+            operation=start_request.operation,
+            headers=headers,
+            request_id=start_request.request_id,
+            callback_url=start_request.callback,
+            inbound_links=[
+                nexusrpc.Link(url=link.url, type=link.type)
+                for link in start_request.links
+            ],
+            callback_headers=dict(start_request.callback_header),
+        )
+        _TemporalStartOperationContext(
+            nexus_context=ctx,
+            client=self._client,
+            info=lambda: Info(task_queue=self._task_queue),
+        ).set()
+        input = LazyValue(
```

Comment:
So to confirm, the lazy value has the serializer and the content, but this seems to have no content and embeds the content in the serializer? Why not use a standard payload serializer and set stream here? I forget what we did in other SDKs here.

Reply:
I was following other SDKs here (Go and Typescript) https://github.com/temporalio/sdk-go/blob/master/internal/internal_nexus_task_handler.go#L471-L473

Reply:
Hrmm, I am a bit confused on those other SDKs too. What if I had a Nexus operation that used the LazyValue's serializer to serialize something? That is meant to work in other Nexus workers but not Temporal Nexus worker? I think it's worth rethinking some of this lazy value stuff.

Reply:
Not blocking this PR.